src/main/java/nl/novi/techiteasycontroller/Model/Television.java

```diff
@@ -0,0 +1,206 @@
+package nl.novi.techiteasycontroller.Model;
+
+import jakarta.persistence.*;
+
+import javax.naming.Name;
+import java.util.List;
+
+@Entity
+@Table(name = "televisions")
+public class Television {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String brand;
+    private String type;
+    private String name;
+    private double price;
+    private double availableSize;
+    private double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private boolean smartTv;
+    private boolean wifi;
+    private boolean voiceControl;
+    private boolean hdr;
+    private boolean bluetooth;
+    private boolean ambiLight;
+    private int originalStock;
+    private int sold;
+
+    @OneToOne
+    private RemoteController remoteController;
+
+
+    @ManyToMany
```

Comment:
Heel netjes

Reply:
Je hebt hieronder ook nog een OneToMany met Televisionwallbracket. Dat is eigenlijk dubbelop, maar ik neem aan dat het een "proof of concept" is